### DISCUSSION TITLE
How to install on Windows?

### DISCUSSION THREAD

**Discussion by ghost** [objeck/objeck-lang#80] [General]
I'm on Windows 7 and unable to install latest Visual Studio. I have UCRT runtime installed. I'm going to use MSYS2 UCRT64. I downloaded the Windows zip. What should I do next? Thanks.

**Comment by objeck** [comment]
64-bit Windows?

**Reply by ghost** [reply]
Yes. Windows 7 64 bit.

**Reply by objeck** [reply]
I cannot get msys2 Clang64 or UCRT64 to function under Windows 7. The system does build under Windows 10 64-bit.
